pallets/loans/src/lib.rs

```diff
@@ -2026,3 +2030,64 @@ impl<T: Config> LoansRateProvider<AssetIdOf<T>> for Pallet<T> {
         None
     }
 }
+
+#[allow(unused_variables)]
+impl<T: Config> LoansOperator<AssetIdOf<T>, AccountIdOf<T>, BalanceOf<T>> for Pallet<T> {
+    fn do_mint(
+        supplier: &AccountIdOf<T>,
+        asset_id: &AssetIdOf<T>,
+        amount: BalanceOf<T>,
```

Comment:
We can remove ref for AssetId because it implemented copy and the copy is cheap I guess